Installation/CHANGES.md

```diff
@@ -29,6 +27,19 @@ Release date: September 2018
     to reflect the real needs of the code (some types and operators were used
     in the code but did not appear in the concepts).
 
+### Polygon Mesh Processing
+
+-  Fix a bug in `isotropic_remeshing()` making constrained vertices missing in the output
+-  Guarantee that constrained vertices are kept in the mesh after calling `isotropic_remeshing()`
+   (and not only constrained points like it was before).
```

Comment:
What do you mean by "constrained points"?

Reply:
vertices on a constraint edge, I guess.

Reply:
In the current version (master), the only thing that is guaranteed if you have constrained vertices is that there will exist in the output some vertices with the same coordinates.

Reply:
Oh ok I understand !

So what about changing the parenthesis by
"(and not only the associated constrained points as it was before)."